engine/src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -84,6 +71,12 @@ trait Backend {
 
   def actorSystem: ActorSystem
 
+  def backendConfigEntry: BackendConfigurationEntry
```

Comment:
I wasn't really addressing that point with these changes, but I certainly wouldn't argue that this class belongs in the backend project.  😄